.github/workflows/build_orca.yml

```diff
@@ -177,11 +177,11 @@ jobs:
         run: |
           cpack -G NSIS
 
-      - name: Pack app
-        if: inputs.os == 'windows-latest'
-        working-directory: ${{ github.workspace }}/build
-        shell: cmd
-        run: '"C:/Program Files/7-Zip/7z.exe" a -tzip OrcaSlicer_Windows_${{ env.ver }}_portable.zip ${{ github.workspace }}/build/OrcaSlicer'
+#      - name: Pack app
+#        if: inputs.os == 'windows-latest'
+#        working-directory: ${{ github.workspace }}/build
+#        shell: cmd
+#        run: '"C:/Program Files/7-Zip/7z.exe" a -tzip OrcaSlicer_Windows_${{ env.ver }}_portable.zip ${{ github.workspace }}/build/OrcaSlicer'
```

Comment:
I guess you have to uncomment these part otherwise the zip file doesn't exist.

Reply:
That's the thing.
If this is uncommented the files get zipped the 1st time and than it gets zipped the second time with this:

``` yml
      - name: Upload artifacts Win zip
        if: inputs.os == 'windows-latest'
        uses: actions/upload-artifact@v4
        with:
          name: OrcaSlicer_Windows_${{ env.ver }}_portable
          path: ${{ github.workspace }}/build/OrcaSlicer_Windows_${{ env.ver }}_portable.zip
          path: ${{ github.workspace }}/build/OrcaSlicer
```

Reply:
It won't with the other change you made.

Reply:
As long as Path points to the dir not the zip it should be fine.